theta_sketch_a_not_b.sql

```diff
@@ -15,8 +15,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-CREATE OR REPLACE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_a_not_b(sketch1 BYTES, sketch2 BYTES, seed INT64) RETURNS BYTES LANGUAGE js
-OPTIONS (library=["$GCS_BUCKET/theta_sketch.js"]) AS R"""
+CREATE OR REPLACE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_a_not_b(sketch1 BYTES, sketch2 BYTES, seed INT64)
+RETURNS BYTES
+LANGUAGE js
+OPTIONS (
+  library=["$GCS_BUCKET/theta_sketch.js"],
+  description = '''Performs the scalar set difference: sketch1 and not sketch2.
+Param sketch1: the first sketch "a" as bytes.
+Param sketch2: the second sketch "b" as bytes.
```

Comment:
I think we need to be consistent and clear just what kind of sketch "bytes" we are expecting: are these Base64 encoded bytes, or raw bytes?  Do they represent a Compressed Compact Sketch, always?

Reply:
as I understand, base64 is implicitly done by the engine. from the user's perspective this is a binary blob. Functions don't not care if the input sketches are compressed or not. We may specify if the output is compressed or not, and maybe even make it configurable, but I am inclined to think that more CPU to reduce IO must be an almost universally good trade-off in systems like this.